vector-im/element-android
# [Device Manager] Render current session (PSG-668)

## Type of change


## Content
Implement current session section as in the new layout.



## Motivation and context
[#6902 ](https://github.com/vector-im/element-android/issues/6902)



## Screenshots / GIFs



|Verified Session (Light)|Verified Session (Dark)|
|-|-|
|![verified_session_light](https://user-images.githubusercontent.com/1254401/185896897-4b364994-5806-44c0-84af-fd0b9ad2dadd.png)|![verified_session_dark](https://user-images.githubusercontent.com/1254401/185896923-66eecf3a-c956-4dca-bb48-4724bc3e31cf.png)|

|Unverified Session (Light)|Unverified Session (Dark)|
|-|-|
|![unverified_session_light](https://user-images.githubusercontent.com/1254401/185896287-0501daf2-77c6-430c-873d-2303f81baca6.png)|![unverified_session_dark](https://user-images.githubusercontent.com/1254401/185896303-8d60995e-9f82-4af5-925f-6bcc4bd35b86.png)|

## Tests

- Enable the new device management feature flag
- Go to "Security & Privacy" settings
- Press the "Show all sessions (V2, WIP)" entry
- See the current session is rendered as in the new layout

## Tested devices

- OS version(s):

## Checklist
